_includes/markdown/Performance.md

```diff
@@ -0,0 +1,374 @@
+
+<h2 id="images-and-media" class="anchor-heading">Images and Media {% include Util/link_anchor anchor="images-and-media" %} {% include Util/top %}</h2>
+
+*   Images should be optimized for Next-Gen formats. JPEG 2000, JPEG XR, and [WebP](https://developers.google.com/speed/webp) are image formats that have superior compression and quality characteristics compared to their older JPEG and PNG counterparts.
+*   Crop images appropriately, you do not need to create a crop for every size but in some cases a few extra crops to handle mobile proportions can be useful.
+*   Images should be served using srcset so that smaller sizes can be shown for smaller viewports.
+*   All images implemented through code should [contain a width and height attribute](https://web.dev/optimize-cls/#images-without-dimensions) this is especially important in avoiding Content Layout Shift issues.
+*   Assets in particular images, should be served through CDN
+*   Images, Videos and iFrames should be lazy loaded. Please note that [WordPress will handle browser level lazy loading](https://make.wordpress.org/core/2021/02/19/lazy-loading-iframes-in-5-7/) using [native lazy load](https://web.dev/native-lazy-loading/). In order for this to take effect the width and height should be set on the tag. 
+*   Hosting videos directly on WordPress should be avoided and can be problematic at scale. 10up recommends a dedicated hosting service such as Brightcove, Vimeo, YouTube, Dailymotion, etc
+
+<h2 id="fonts" class="anchor-heading">Fonts {% include Util/link_anchor anchor="fonts" %} {% include Util/top %}</h2>
+
+*   Fonts used across the site should be [preloaded](https://web.dev/codelab-preload-web-fonts/). If the fonts are not self hosted then ensure to include the _`crossorigin`_ attribute.
+*   Whenever available, WOFF2 font formats should be used for better compression with a WOFF fallback.
+*   Font files should be minimized (don’t load styles you don’t need)
+*   Fonts should either be served locally or from a single foundry (don’t mix Google fonts and typekit, pick one).
+
+<h2 id="javascript-and-css" class="anchor-heading">JavaScript and CSS {% include Util/link_anchor anchor="javascript-and-css" %} {% include Util/top %}</h2>
+
+*   Use “Mobile First” build concepts so the browser doesn’t have to render extra CSS at load time.
+*   All JavaScript and CSS should be minified.
+*   Standalone site features should be broken off into isolated entry points so we don’t have to load more CSS/JS on pages that will never use it. 
+*   No unnecessary libraries should be loaded. If something is not being used, do not load it, if only a small part of a library is needed, consider custom building or loading only the part that's needed.
```

Comment:
We should tighten this up - its not always as clear as just forgoing unecessary libraries. Some 3rd-party scripts will embed libraries beyond our control and we cant do much about that.